docs/program-setup/bare_metal/mbed2_porting.md

```diff
@@ -0,0 +1,109 @@
+# Porting an Mbed 2 target to Mbed OS 6 bare metal
+
+The Mbed OS bare metal profile removes the RTOS and provides fewer features compared to Mbed OS 6. This profile offers the same functionality as Mbed 2 and allows targets to access features that we've added to Mbed OS 5 and 6. This document describes how to configure an Mbed 2 target to run in Mbed OS 6 bare metal and how to validate the port.
```

Comment:
> The Mbed OS bare metal profile removes the RTOS and provides fewer features compared to Mbed OS 6.

Bare metal is a profile to Mbed OS 6 so the comparison does not quite make sense. Perhaps just stop at «features ».

Reply:
> we've added to Mbed OS 5 and 6.

How about « we have added to more recent versions of Mbed OS »?

Reply:
> how to configure an Mbed 2 target to run in Mbed OS 6 bare metal 

How about « how to configure an Mbed 2 target to run with Mbed OS using the bare metal profile »?

Reply:
> > The Mbed OS bare metal profile removes the RTOS and provides fewer features compared to Mbed OS 6.
> 
> Bare metal is a profile to Mbed OS 6 so the comparison does not quite make sense. Perhaps just stop at «features ».

That does not work for me. If it has less features then we should say compared to what.

Reply:
I will rephrase;  I would like to introduce the main two parts of this page: `Configuring your target` and `Validating the port`.